build/screenshots/Jenkinsfile

```diff
@@ -0,0 +1,54 @@
+pipeline {
+  agent any
+
+  options {
+    buildDiscarder(logRotator(numToKeepStr: '30', artifactNumToKeepStr: '5'))
+  }
+
+  triggers {
+    cron '@midnight'
+  }
+
+  stages {
+    stage('Screenshot') {
+      steps {
+        script {
+          catchError(buildResult: 'UNSTABLE', stageResult: 'UNSTABLE') {
+            docker.image('mcr.microsoft.com/playwright:v1.43.0-jammy').inside {
+              sh 'npm install'
+              dir ('playwright') {
+                sh 'npm run webtest:screenshot'
+              }
+            }
+          }
+          archiveArtifacts artifacts: '**/playwright/target/screenshots/**', allowEmptyArchive: true
+          withChecks('IT WebTest') {
+            junit testDataPublishers: [[$class: 'StabilityTestDataPublisher']], testResults: 'playwright/**/report.xml'
+          }
+        }
+      }
+    }
+    stage('Deploy') {
+      when {
+        expression { isReleaseOrMasterBranch() && currentBuild.currentResult == 'SUCCESS' }
+      }
+      steps {
+        script {
+          docker.image('maven:3.8.6-eclipse-temurin-17').inside {
+            maven cmd: '-ntp -f playwright/tests/screenshots clean deploy -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn'
+          }
+          archiveArtifacts 'playwright/tests/screenshots/target/form-editor-screenshots-*.zip'
+        }
+      }
+    }
+  }
+  post {
+    always {
+      discoverGitReferenceBuild()
```

Comment:
copied: not sure what it is for :thinking:

Reply:
Not sure, for detect if git changes happened?